homeassistant/components/netatmo/light.py

```diff
@@ -0,0 +1,155 @@
+"""Support for the Netatmo camera lights."""
```

Comment:
Yes it can. This was added because exposing the camera lights as services did not seem to fit IMO. Of course I can make this a separate PR if you wish.

Reply:
Is that an existing feature that is moved to the light platform?

Reply:
Yes. See https://github.com/home-assistant/home-assistant/pull/27970/files

Reply:
Now that the functionality to turn the lights on/off has been removed due to guideline violating implementation details I'll discuss to get this into the upstream module. Functionality will be added back in as soon as the upstream changes are merged.

Reply:
Should we remove the platform for now? If the entity can only report state it doesn't fit the light platform.

Reply:
That is probably right. I'll add this back in once the API documentation has been fixed and it can be implemented properly upstream.